prow/cmd/admission/dev.yaml

```diff
@@ -0,0 +1,90 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: prow-admission
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: prow-admission
+  template:
+    metadata:
+      labels:
+        app: prow-admission
+    spec:
+      serviceAccount: prow-admission
+      containers:
+      - name: admission
+        image: gcr.io/k8s-testimages/admission:latest  # Note: not gcr.io/k8s-prow for dev
+        imagePullPolicy: Always  # Good practice for dev/debugging, bad for prod
```

Comment:
Great for prod.. you either have node-local caching for deterministic image tags or always update your floating tag :)

Reply:
This is not meant to be a production deployment. I will follow with on in a different PR.

Reply:
Steve, what are you saying is great for prod?

The intent of this (copied) comment is to note that immutable tags work better than mutable ones, as with a `latest` tag, updating latest will not cause kubernetes to reload the pods for this deployment.

Applying a change of `latest-1` to `latest-2` will cause the deployment contorller to reload the pods.

Reply:
When someone does use a mutable tag in their deployment, they intend for their deployment to float with the tag and they _want_ updates to the mutable tag to propagate to their deployments

Reply:
My point is that mutable produce arbitrary behavior and thus are bad for production. AKA when you apply a new tag to the deployment, you'll know this will result in a new pod that uses that tag.

When you overwrite a tag with new source, kubernetes continues using the old source until the pod happens to die, at which the new pod may use either the new or old source.

Setting an always pull policy addresses the latter half (the new pod will always use the new source) but not the first (does not cause kubernetes to recreate the pod).

Thus, mutable tags are bad for prod.